script/zonemaster_backend_testagent

```diff
@@ -87,13 +87,14 @@ sub main {
     my $self = shift;
 
     my $caught_sigterm = 0;
-    local $SIG{TERM} = \&catch_sigterm;
-    sub catch_sigterm {
-        $SIG{TERM} = \&catch_sigterm;
+    my $catch_sigterm;
+    $catch_sigterm = sub {
```

Comment:
Why not combine the two lines into
```
my $catch_sigterm = sub {
```

Reply:
Because the variable is referenced inside the closure.